### PR TITLE
chore: update issue and PR templates

### PR DESCRIPTION
**Pull request by sedghi**
### PR Checklist

- [ ] Brief description of changes
- [ ] Links to any relevant issues
- [ ] Required status checks are passing
- [ ] User cases if changes impact the user's experience
- [ ] `@mention` a maintainer to request a review

[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch

### CODE REVIEW

**Review by wayfarer3130 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/---feature-request.md [hunk 1]
```diff
@@ -2,19 +2,23 @@
...
-> out to a community member that offers
+> [contributing it](https://v3-docs.ohif.org/development/contributing) yourself!
+> Or reach out to a community member that offers
 > [consulting services](https://docs.ohif.org/help.html#paid--commercial).
+-->
 
-## Request
+### Request
 
 **What feature or change would you like to see made?**
```
Comment (jbocce) [inline .github/ISSUE_TEMPLATE/---feature-request.md:23]: @sedghi  consider  adding information like the following, not a big deal, just something to consider.

- Please include as much detail as possible including possibly mock up screen shots, workflow or logic flow diagrams etc.
- Discuss if and how the requested feature interacts with existing features

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,14 +1,59 @@
...
-- [ ] `@mention` a maintainer to request a review
+### Context
 
 <!--
-  Links
-  -->
+Provide a clear explanation of the reasoning behind this change, such as:
+- A link to the issue being addressed, using the format "Fixes #ISSUE_NUMBER"
+- An image showing the issue or problem being addressed
+- Error logs or callStacks to help with the understanding of the problem
```
Comment (jbocce) [inline .github/pull_request_template.md:18]: Is this really necessary? Shouldn't this type of information exist already in the original issue?

Comment (sedghi) [reply .github/pull_request_template.md:18]: I don't see why not?

Comment (sedghi) [reply .github/pull_request_template.md]: done

**Pull request merged**